src/Import/AbstractImport.php

```diff
@@ -32,8 +32,25 @@ protected function addEventContent(CalendarEventsModel $objEvent, array $eventco
     {
         $step = 128;
 
+        $columns = ['ptable=? AND pid=?'];
+        $values = ['tl_calendar_events', $objEvent->id];
+        $contents = ContentModel::findBy($columns, $values);
+        $contentDictionary = [];
+
+        foreach ($contents as $content) {
+            $key = $content->sorting;
+            if (isset($contentDictionary[$key])) {
+                $key = uniqid('', true);
+            }
+            $contentDictionary[$key] = $content;
+        }
+
         foreach ($eventcontent as $content) {
-            $cm = new ContentModel();
+            if (isset($contentDictionary[$step])) {
+                $cm = $contentDictionary[$step];
+                unset($contentDictionary[$step]);
+            }
+            $cm = $cm ?? new ContentModel();
```

Comment:
Please change this line to

`$cm ??= new ContentModel();`

Then the checks should be successful